### DISCUSSION TITLE
what is the procedure to change the public  ip address?

### DISCUSSION THREAD

**Discussion by inetactief** [AzuraCast/AzuraCast#5123] [Q&A]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #7870205 (2022-02-04 2:30) • Docker • PHP 8.1

### What happened?

Hello,

We like to move the server and we get a new ip adress what is the procedure for that? 
Can not find it in the docs https://docs.azuracast.com/en

**Comment by renatopinto89** [comment]
The public IP is assigned to you by your server hosting provider. It has nothing to do with AzuraCast. Usually if you move providers/servers, a different IP will be assigned to that server.

**Comment by inetactief** [comment]
Sorry I wasn't clear. I understand that this has to be done with the provider. 
The question is, do I still need to add/change something in the Azuracast software or does that automatically work on the new ip address? 
( and i must change the dns A records)

**Comment by renatopinto89** [comment]
If you're running a domain and you keep the same domain, if you restore from a backup on your new AzuraCast installation, everything should work fine. Just make sure you point the domain to the new IP address on a A Record

**Reply by inetactief** [reply]
So I change the public ip address on the server (in my case Centos 7) I change the DNS name to new ip address and reboot? Furthermore, do I not have to make any adjustments to the software? 
Everything works on DNS name? And it remains the same.
Is that correct?
